### ISSUE TITLE
Incorrect mapping of Peter Bird zones to a-grid file

### ISSUE THREAD

**Issue by pmpowers-usgs** [usgs/nshmp-haz-fortran#12]
Reported by @AHajiSoltani and @JMAltekruse
Also noted by @treyapel and @nileshshome

`WUS_zones_PB.in` specifies:

```
0       !use hazgridXnga4. with wtgrid for M>6.5 Nov 14 2007
24.6 50. 0.05     EXTmapC.in for NGA firm rock Agrids new Sept 20 2007
-125. -100. 0.05
760. 2.
2 1. 0.0 1.0 5. 1.0 0.0  !2 depths
0.0 0.0 1.0 !weights to ss rev normal respectively
1. 300.
24.6 50. 0.1     !EXTENDED source grid 
-125. -100. 0.1
0.8 6.5 7.5 .1 3.
1 0 0 0
../conf/WUS/gridded/GR_DOS/agrd_zonesPB.out
```

`agrd_zonesPB.out` is a larger file (1MB) indicating it is a 0.05 degree discretized a-grid, but it is parsed as a 0.1 degree discretized grid. This grid source is used to represent diffuse faulting and is combined with other Peter Bird fault sources assuming a 0.05 degree calculation grid. There is a smaller 0.1 degree a-grid file in the GR_DOS directory that was probably being used at one point; when the larger, high-resolution file was adopted instead, this input file was not updated. The net result is these sources are not occurring where they should be. See attached image.
![wus_zonespb](https://cloud.githubusercontent.com/assets/2379604/11042130/ac34a214-86d2-11e5-9a31-043dc39b8c34.jpg)

**Comment by pmpowers-usgs** [comment]
For consistency with fault sources (the hazard for which is computed at 0.05°), we:
- Keep `conf/WUS/gridded/GR_DOS/agrd_zonesPB.out` (0.1°)
- Delete `conf/WUS/gridded/GR_DOS/agrd_bird_zones.out` (0.05°)
- Update `conf/WUS/gridded/WUS_zones_PB.in` source grid spec to 0.05° discretization

**Comment by pmpowers-usgs** [comment]
It turns out that once both the 0.1° and 0.05° are correctly specified in WUS_zones_PB.in, a third zone, corresponding to the Hite Fault in SW Washington is also included. This is at odds with Figure 53 in the [OFR](http://pubs.usgs.gov/of/2014/1091/). The Hite Fault is represented in all fault files; need to confirm (@haller-usgs) that the faults are reflecting the full slip rate and then can zero out a-grids.

![hitefault](https://cloud.githubusercontent.com/assets/2379604/11284324/5c470d68-8ec6-11e5-8667-5e8e136da8a4.jpg)

**Comment by pmpowers-usgs** [comment]
closed by #24
